metrics/api/v1/model_handlers.go

```diff
@@ -33,7 +33,26 @@ import (
 // when manager.model has not been initialized.
 var errModelNotActivated = errors.New("the model is not activated")
 
-var metricNamesConversion = map[string]string{
+var supportedMetrics = map[string]struct{}{
```

Comment:
So what do you want to return in `/metrics` requests? Only available (collected) metrics for the requested resource?